src/andromede/input_converter/src/cc_configuration/batteries.yaml

```
@@ -0,0 +1,119 @@
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+# Configuration file to convert a complex modelling artefact in an Antares study into an Andromede model.
+
+loop: 
+    
+  iteration_key: zone #Key on which we iterate (for a given list of zone, we want to execute a loop : for zone in list_zone)
+  #In the following, the notation $zone$ means the value of the string variable "zone"
+  andromede-model: antares-historic.short-term-storage #we want to create components of such a model, with id being indexed by the iteration_key
+  andromede-component-id : battery_$zone$
+
+      
+  andromede-model-parameters-to-set-a-priori:
+    #TODO : we still have to see if the following parameters can be set a priori or have to be read in the Antares study,during the conversion.
+    #- id: lower_rule_curve
+    #- id: lower_rule_curve
+    #- id: p_max_injection_modulation  # Read in p_max_injection 
+    #- id: p_max_withdrawal_modulation # Read in p_max_withdrawal
+    
+    
+    - id: initial_level
+      time-dependent: false
+      scenario-dependent: false
+      value: -1.0 #TODO : The initial stock level constraint must be disabled or deleted in the Andromede ST storage model
+              
+    - id: inflows #No inflows in a battery model!
+      time-dependent: false
+      scenario-dependent: false
+      value: 0.0
+
+          
+    
+  antares-coupling-constraints-to-visit:
+      
+    - id : batteries_$zone$
+      name: batteries_$zone$
+      to-delete: True #This flag means that the constraint should be deleted from the Antares Study, if hybrid mode is targetted
+      parameters-to-store:
+        - id-target-andromede-model-parameter: efficiency_injection #target parameter in the andromede-model
+          location-in-legacy-study: $zone$%z_batteries #where to read the parameter in the Antares coupling constraint
+          multiplication-factor: -1 # The parameter we read in Antares coupling constraint should be multiplied by -1 to create target param
+        - id-target-andromede-model-parameter: efficiency_withdrawal #there are different naming conventions between Antares CC model for batteries and the short-term storage model
+          #=> therefore there is an intervertion between injection and withdrawal (this is no mistake)
+          location-in-legacy-study: $zone$.$zone$_batteries_inj 
+          multiplication-factor: 1
```

Instead of giving the Antares components to iterate through, we could specify directly the location of the Andromede model parameters :
```suggestion
  antares-coupling-constraints-to-visit:
      
    - id: efficicency_injection
      time-dependent: false
      scenario-dependent: false
      value: batteries_$zone$::$zone$%z_batteries
      multiplication_factor: -1

    - id: efficicency_withdrawal
      time-dependent: false
      scenario-dependent: false
      value: batteries_$zone$::$zone$.$zone$_batteries_inj 
```
And add a field a the end to list the legacy components to delete